docker/html/index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
+
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
+    <head>
+        <title>Fileglancer</title>
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+        <style type="text/css">
+            /*<![CDATA[*/
+            body {
+                background-color: #fff;
+                color: #000;
+                font-size: 0.9em;
+                font-family: sans-serif,helvetica;
+                max-width: 80em;
+                padding: 0 10px;
+                margin: 1em auto 1em;
+                position: relative;
+            }
+            h1 {
+                text-align: center;
+                margin: 0;
+                padding: 0.6em 2em 0.4em;
+                background-color: #303030;
+                color: #fff;
+                font-weight: normal;
+                font-size: 1.75em;
+            }
+            .banner {
+                text-align: center;
+                background-color: #571e12;
+                font-size: 1.3em;
+                font-weight: bold;
+                color: #fff;
+                margin-top: 1em;
+                padding: 0.6em;
+                border: 1px solid #000;
+            }
+            .banner :link,:visited,a:hover {
+                color: #fa785c;
+            }
+            h3 {
+                text-align: center;
+                background-color: #ddd;
+                font-size: 1.1em;
+                font-weight: bold;
+                color: #333;
+                margin: 0;
+                padding: 0.5em;
+                border-bottom: 2px solid #000;
+            }
+            .box {
+                margin-top: 1em;
+                border: 2px solid #000;
+            }
+            .content {
+                padding: 1em 5em;
+            }
+            /*]]>*/
+        </style>
+    </head>
+
+    <body>
+        <h1>Welcome to Fileglancer</h1>
```

Comment:
Do we need an index.html? Can we just redirect / to /jupyter?

Reply:
Yeah, let's take people directly to the login page.

Reply:
Done